### PR TITLE
W-13600668:  Seamless setup mule management

### PR DESCRIPTION
**Pull request by fermujica**

### CODE REVIEW

File: modules/ROOT/pages/manage-mule-runtime.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+This warning happens when you open Anypoint Code Builder without a Mule Runtime engine configured. It's possible that one or both of the *Mule Home* and *Mule Runtimes Directory* are misconfigured, or empty. To troubleshoot this:
+
+. Navigate to your Anypoint Code Builder view, and select the *Open ACB Settings* icon.
++
+image::open-settings.png[]
+. Locate the *Mule > Runtime: Mule Home* and *Mule > Runtime: Mule Runtimes Directory* properties:
++
+image::mule-runtimes-settings.png[]
+. Ensure that both fields are not empty, and that the configurations point to an actual ZIP file containing Mule runtime.
```
Comment (greobasco) [inline modules/ROOT/pages/manage-mule-runtime.adoc]: **ZIP file** should be replaced by **folder**

File: modules/ROOT/pages/manage-mule-runtime.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+This error happens when you attempt to use a feature that requires a working Mule Runtime engine configured. For example, attempting to use auto-complete, or run a debug session. It's possible that one or both of the *Mule Home* and *Mule Runtimes Directory* are misconfigured, or empty. To troubleshoot this:
+
+. Navigate to your Anypoint Code Builder view, and select the *Open ACB Settings* icon.
++
+image::open-settings.png[]
+. Locate the *Mule > Runtime: Mule Home* and *Mule > Runtime: Mule Runtimes Directory* properties:
++
+image::mule-runtimes-settings.png[]
+. Ensure that both fields are not empty, and that the configurations point to an actual ZIP file containing Mule runtime.
```
Comment (greobasco) [inline modules/ROOT/pages/manage-mule-runtime.adoc]: **ZIP file** should be replaced by **folder**

Comment (dukesphere) [reply modules/ROOT/pages/manage-mule-runtime.adoc]: @greobasco Is it the full path, including the folder that contains Mule, or is it to a named folder?

Comment (dukesphere) [inline modules/ROOT/pages/manage-mule-runtime.adoc]:
```suggestion
. Provide the full path to the folder that contains Mule runtime. This field must not be empty. 
```

File: modules/ROOT/pages/manage-mule-runtime.adoc (hunk 1 above)
Comment (dukesphere) [inline modules/ROOT/pages/manage-mule-runtime.adoc]:
```suggestion
. Provide the full path to the folder that contains Mule runtime. This field must not be empty. 
```

File: modules/ROOT/pages/manage-mule-runtime.adoc (hunk 2 above)
Comment (greobasco) [reply modules/ROOT/pages/manage-mule-runtime.adoc]: Yes, the full path. Examples can be viewed in the picture:
https://dev-docs-internal.kqa.msap.io/beta-code-builder/anypoint-code-builder/_images/mule-runtimes-settings.png

The **Mule Runtimes Directory** contains the full path to the folder that contains the Mule runtimes:
i.e: `/home/mule-dx/muleRuntimes`

The **Mule Home** contains the full path to the specific runtime: 
i.e: `/home/mule-dx/muleRuntimes/mule-enterprise-standalone-4.4.0-20230417/`

**Review by greobasco — APPROVED** [review]

File: modules/ROOT/pages/manage-mule-runtime.adoc [hunk 3]
```diff
@@ -1,46 +1,44 @@
-= Manage Mule Runtime in Code Builder
+= Mule Runtime Not Found or Not Installed Errors
 
-Anypoint Code Builder bundles an instance of Mule Runtime engine for the implement and integration tasks. You must always have a Mule runtime configured to be able to use the features that enable you to develop Mule applications.
+Anypoint Code Builder bundles a version 4 instance of Mule runtime engine (Mule) to run your integrations and implementations.  
 
-Code Builder has two properties to manage the embedded Mule Runtime engine instance:
+Do not change the default locations of Mule are unless you receive an error that Mule is not found or installed. If such errors occur, 
+the following properties enable you to change the location:
```
Comment (dukesphere) [inline modules/ROOT/pages/manage-mule-runtime.adoc]:
```suggestion
the following properties enable you to change the path to Mule:
```

File: modules/ROOT/pages/manage-mule-runtime.adoc [hunk 4]
```diff
@@ -1,46 +1,44 @@
...
-[WARNING]
---
-Changing the *Mule Home* and *Mule Runtimes Directory* properties requires advanced knowledge of Mule runtime.
 
-Changing the default values causes unexpected errors.
---
+== Mule Runtime Is Not Installed, You Must Configure a Mule Runtime to Enable This Feature
 
-== Troubleshooting Embedded Mule Runtime Errors
+This error occurs if *Mule Home* and *Mule Runtimes Directory* are misconfigured or empty and feature requires a running Mule instance.
```
Comment (dukesphere) [inline modules/ROOT/pages/manage-mule-runtime.adoc]:
```suggestion
This error occurs if *Mule Home* and *Mule Runtimes Directory* are misconfigured or empty and a feature requires a running Mule instance. 
```

File: modules/ROOT/pages/manage-mule-runtime.adoc [hunk 5]
```diff
@@ -1,46 +1,44 @@
...
-Code Builder has two properties to manage the embedded Mule Runtime engine instance:
+Do not change the default locations of Mule are unless you receive an error that Mule is not found or installed. If such errors occur, 
+the following properties enable you to change the path to Mule:
 
-* *Mule Home*: The full path that points to the ZIP file of the Mule Runtime instance. +
-Default value: `/Users/<user-name>/muleRuntimes/<mule-runtime-zip-file>`.
-* *Mule Runtimes Directory*: The default location in which all Mule runtime engine ZIP files are downloaded. +
+* *Mule Runtimes Directory*: The default location of the Mule runtime engine installation. +
 Default value: `/Users/<user-name>/muleRuntimes/`.
+* *Mule Home*: The full path that points to the root files of the Mule Runtime instance. +
```
Comment (dukesphere) [inline modules/ROOT/pages/manage-mule-runtime.adoc]:
```suggestion
* *Mule Home*: The full path that points to the root files of the Mule instance. +
```

File: modules/ROOT/pages/manage-mule-runtime.adoc [hunk 6]
```diff
@@ -1,46 +1,44 @@
...
+Anypoint Code Builder bundles a version 4 instance of Mule runtime engine (Mule) to run your integrations and implementations.  
 
-Code Builder has two properties to manage the embedded Mule Runtime engine instance:
+Do not change the default locations of Mule are unless you receive an error that Mule is not found or installed. If such errors occur, 
+the following properties enable you to change the path to Mule:
 
-* *Mule Home*: The full path that points to the ZIP file of the Mule Runtime instance. +
-Default value: `/Users/<user-name>/muleRuntimes/<mule-runtime-zip-file>`.
-* *Mule Runtimes Directory*: The default location in which all Mule runtime engine ZIP files are downloaded. +
+* *Mule Runtimes Directory*: The default location of the Mule runtime engine installation. +
```
Comment (dukesphere) [inline modules/ROOT/pages/manage-mule-runtime.adoc]:
```suggestion
* *Mule Runtimes Directory*: The default location of the Mule installation. +
```

**Review by greobasco — APPROVED** [review]
LGTM

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**